app/timetables/models.py

```diff
@@ -175,3 +175,27 @@ def __str__(self):
 
     class Meta:
         unique_together = ('timetable', 'cycle_day', 'meal', 'meal_option')
+
+
+class Event(TimestampMixin):
+    """
+    Model representing a date or range of dates to which a
+    specific action can be assigned.
+    """
+
+    timetable = models.ForeignKey(Timetable)
+    start_date = models.DateField()
+    end_date = models.DateField()
+
+    def clean(self):
+        if self.start_date >= self.end_date:
+            raise ValidationError(_('start_date must be less than end_date.'))
```

Comment:
Since people will use the admin interface, let us use a more readable validation error instead of one with variable names.